### ISSUE TITLE
Stil need a long time to synthesize a new speech

### ISSUE THREAD

**Issue by liangshuang1993** [xcmyz/FastSpeech#9]
I've trained a new model, and write a script to synthesize new speech using 
```
mel_output, mel_output_postnet = model(src_seq, src_pos)
```

But it need about 4-6 seconds, is this an expected result? Thanks!

**Comment by Liujingxiu23** [comment]
How about your wave synthesized?  I synthesized wavs using model checkpoint "checkpoint_40000.pth.tar". The quality is bad.

**Comment by xcmyz** [comment]
Total loss is as follows (I cut top 30000):
![loss_one](https://user-images.githubusercontent.com/39415743/59324680-8b72e000-8d12-11e9-8131-c62dcb86f8c7.jpg)

**Comment by xcmyz** [comment]
I will put result wav here soon.

**Comment by liangshuang1993** [comment]
@xcmyz  Can you tell me how long do you need to synthesis a speech? Thanks!

**Comment by Liujingxiu23** [comment]
@xcmyz  Thank you for your reply. How many iters the model need to converge ? I use the LJSpeech dataset

**Comment by liangshuang1993** [comment]
@Liujingxiu23 My loss is about 0.2, not good either. How long do you need to synthesis a new speech?

**Comment by Liujingxiu23** [comment]
mel_output, mel_output_postnet = model(src_seq, src_pos)
mel = mel_output_postnet[k].detach().cpu().numpy()
wav = audio.inv_mel_spectrogram(mel.T)

For batch-size=10, seq_len=75, using one GPU, the time spend:
1. to generate mel and mel_postnet  : 0.159 sec
1. to generate mel, mel_postnet and wavs:  37.625 sec

**Comment by liangshuang1993** [comment]
@Liujingxiu23  Thanks, I just found I calculate time in a wrong way, to generate mel and mel_postnet, it takes about 0.05 using P100.

**Comment by Liujingxiu23** [comment]
@xcmyz I tried your laest code, the acoustic quality improve much, early the same as tacotron2 I think.
The TTS corpus I use is chinese, and I keep the default hparams setting.
My loss seems not as good as yours, the postnet-mel-loss converge to about 0.5, the duration loss about 0.8. I don not know why? 
By the way the pronunciation as well as the tone is not that good. For example, in same wavs "zhang" read like "zhan"， “tao3” read like “tao2” ，why this happens? Do you have any suggest to solve this problem?
